src/actions/PermissionsActions.js

```diff
@@ -63,18 +63,16 @@ const getImage = async (dispatch, imageSourceType) => {
 
         result = await ImagePicker.launchImageLibraryAsync({
             allowsEditing: true,
-            // aspect: [4, 4],
             base64: true,
-            quality: 0.5
+            quality: 0.4
         });
     } else if (imageSourceType === imageSourceTypes.CAMERA) {
         console.log("In the CAMERA area: ",imageSourceType);
 
         result = await ImagePicker.launchCameraAsync({
             allowsEditing: true,
-            // aspect: [4, 4],
             base64: true,
-            quality: 0.5
+            quality: 0.4
```

Comment:
We're gonna have to keep playing with this, because we're passing around raw base64 data for images. The quality is from 0 to 1. I knocked it down another 10% since I ran into a firebase storage issue with it at 0.5. Gonna have to keep an eye on this.